client/src/components/Slider/Slider.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import "./Slider.css";
+
+
+const Slider = (props) => {
+        let cards = props.cards; //all cards
+        let [end, setEnd] = useState(3); //first three cards
+        let [beg, setBeg] = useState(0); //first three cards
```

Comment:
This is cool, but we're not using hooks in this app. If this is a component that needs state, can you use a class component?